level-3/README.md

```diff
@@ -0,0 +1,25 @@
+# karma-clone
+## Level 3
+
+Now let's make this website responsive! 
+
+### Set up
+
+You can choose to add your code for Level 3 to the same branch you were using for the previous levels, or to create a new 'level-3' branch. 
+
+# Instructions
+
+- Use the techniques you've learned in class to make both the homepage and the Order page look good on mobile. 
+- You'll find the mobile designs inside the 'Level 3' folder. 
+- You can use 1 or 2 breakpoints, it's up to you (if you're not sure, start with 768px - it's a classic one as it's the width of an iPad and a good point to switch from a mobile layout to a desktop layout).
+
```

Comment:
```suggestion
- You might have to change some of your existing HTML and CSS code so that your webpage can match the mobile and desktop designs

```
I wonder if some students might work on this task thinking that they're only to add CSS within breakpoints and leave the rest of their code untouched. If so, we could explicitly say they might have to modify their existing code?

Reply:
👍